docs/05-PipelineLaunchOptions.md

```diff
@@ -0,0 +1,99 @@
+the automated quality control and MRI protocol checks performed by the pipeline 
+should remain identical.
+
+It is also worth mentioning that all the options illustrated here assume that 
+the candidate and visit are already registered in the LORIS database.
+
+
+In the next three sub-sections, the three launch options illustrated in the 
+figure above will be briefly highlighted and the exact command needed to launch 
+the pipeline shown.
+Details about the scripts themselves can be found in the [Scripts](04-Scripts.md)
+section.
+ 
+## 5.1 Option 1
+
+Triggering the pipeline is done from the `/data/$PROJECT/bin/mri` directory as 
+follows:
+
+```
+uploadNeuroDB/imaging_upload_file.pl -profile prod -upload_id $UploadIDID /PATH/TO/UPLOAD/PSCID_CandID_VisitLabel_OptionalSuffix.zip -verbose
+```
+
+where `$UploadID` is the number corresponding to the `UploadID` column in the 
+Imaging Uploader table, and `/PATH/TO/UPLOAD/` is typically the 
+`/data/incoming/` directory.
+
+This is a typical option for a project prospectively collecting data with 
+
+1. multiple sites involved, 
+2. a designated user per site for collecting scans from the scanner console and 
+uploading to LORIS, and 
+3. a project's imaging specialist monitoring new uploads, and launching the 
+insertion pipeline manually, for every new `UploadID` separately. 
+
+## 5.2 Option 2 
+
+Triggering the pipeline from the `/data/$PROJECT/bin/mri` directory can also be 
+achieved as follows: 
+
+```
+uploadNeuroDB/imaging_upload_file_cronjob.pl -profile prod -verbose
+```
+                                
+The script `imaging_upload_file_cronjob.pl` does not require a specific 
+`$UploadID` as an argument because it automatically cycles through 
+I<all new uploads> on which the pipeline has not been triggered yet, and 
+launches it. As such, this option is used in similar scenarios as Option 1, 
+but with the added advantage of not needing to continuously monitor for new 
+uploads (the script can be added to the system's crontab to launch at 
+pre-scheduled times, as per the project's requirements).
+
+
+
+## 5.3 Option 3
+
+The insertion pipeline can then be triggered using the command:
+```
+./batch_uploads_imageuploader -profile prod < scans_list.txt >log_batch_imageuploader.txt 2>&1 
+
+```
+
+This is an option that addresses retrospectively collected data where uploading 
+hundreds of scans, one scan at a time, using the LORIS Imaging Uploader a 
+user-friendly interface is impractical. It is also the option of choice for 
+prospective studies that want to benefit from tracking scans through the Imaging 
+Uploader while automating the upload and insertion process without a user/GUI 
+interface interaction. 
+
+In this case, the scans should be transferred to the LORIS-MRI filesystem (a 
+path such as `/data/incoming/`), and on input text file `scans_list.txt` 
+provided, listing one line pr dataset (example below).
+The following fields are expected, separated by spaces:
```

Comment:
would rephrase this to:

"Each line in that text file should include the following information separated by spaces:"